tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/async/BasicAsyncOperationsTest.java

```diff
@@ -91,8 +91,7 @@ public void testAsyncNonNativeBasic8() {
 
     @Test
     public void testAsyncNonNativeBasic9() {
-        Object returns = BRunUtil.invoke(result, "testAsyncNonNativeBasic9", new Object[0]);
-        Assert.assertEquals(returns, 7L);
+        BRunUtil.invoke(result, "testAsyncNonNativeBasic9", new Object[0]);
```

Comment:
Why we remove assertion here

Reply:
The assertion is moved to Ballerina side to assert error details.